src/types.ts

```diff
@@ -490,6 +490,8 @@ export interface PupilFieldsData {
 
 export type PupilFieldsResponse = ClassChartsResponse<PupilFieldsData, []>;
 
+export type ChangePasswordResponse = ClassChartsResponse<[], []>;
```

Comment:
Does the API not give any feedback to whether the request was successful? And any meta value?

Reply:
The api returns this { success: 1, data: [], meta: [] }

Reply:
Ah fair enough